frontend/src/app/search/actions.ts

```diff
@@ -5,7 +5,7 @@ import { APISearchFetcher } from "../../services/searchfetcher/APISearchFetcher"
 import { MockSearchFetcher } from "../../services/searchfetcher/MockSearchFetcher";
 import { fetchSearchOpportunities } from "../../services/searchfetcher/SearchFetcher";
 
-const useMockData = false;
+const useMockData = true;
```

Comment:
I'll pull this out to an env variable in `.env.development` in a separate PR

Reply:
Reverted this